timed/tracking/views.py

```diff
@@ -236,6 +211,41 @@ def export(self, request):
             "task__project__cost_center",
         )
         queryset = self.filter_queryset(queryset)
+        queryset = queryset.annotate(
+            cost_center=Case(
+                # Task cost center has precedence over project cost center
+                When(
+                    task__cost_center__isnull=False, then=F("task__cost_center__name")
+                ),
+                When(
+                    task__project__cost_center__isnull=False,
+                    then=F("task__project__cost_center__name"),
+                ),
+                default=Value(""),
+                output_field=CharField(),
+            )
+        )
+        queryset = queryset.annotate(
+            billing_type=Case(
+                When(
+                    task__project__billing_type__isnull=False,
+                    then=F("task__project__billing_type__name"),
+                ),
+                default=Value(""),
+                output_field=CharField(),
+            )
+        )
+        if (
+            settings.REPORTS_EXPORT_MAX_COUNT > 0
+            and queryset.count() > settings.REPORTS_EXPORT_MAX_COUNT
+        ):
+            return Response(
+                "Your request exceeds the maximum allowed entries ({0})".format(
+                    settings.REPORTS_EXPORT_MAX_COUNT
+                ),
+                status=status.HTTP_400_BAD_REQUEST,
+            )
```

Comment:
The message should be added to `django.po`. And maybe not only include how many entries are allowed, but also how many entries were included in the query.

Reply:
Haha didn't even realize there are "translations". Nobody touched it in more than 2 years so there's some more messages to check-in... :sweat_smile:

Reply:
:sweat_smile: 

Could you please add the missing translations strings in a separate commit.

Reply:
done